src/Microsoft.ML.Transforms/FourierDistributionSampler.cs

```diff
@@ -5,205 +5,252 @@
 using System;
 using Microsoft.ML;
 using Microsoft.ML.CommandLine;
-using Microsoft.ML.EntryPoints;
+using Microsoft.ML.Data;
 using Microsoft.ML.Internal.Utilities;
 using Microsoft.ML.Model;
+using Microsoft.ML.Numeric;
 using Microsoft.ML.Transforms;
 
-[assembly: LoadableClass(typeof(GaussianFourierSampler), typeof(GaussianFourierSampler.Options), typeof(SignatureFourierDistributionSampler),
-    "Gaussian Kernel", GaussianFourierSampler.LoadName, "Gaussian")]
+[assembly: LoadableClass(typeof(GaussianRngGenerator), typeof(GaussianRngGenerator.Options), typeof(SignatureRngGenerator),
+    "Gaussian Kernel", GaussianRngGenerator.LoadName, "Gaussian")]
 
-[assembly: LoadableClass(typeof(LaplacianFourierSampler), typeof(LaplacianFourierSampler.Options), typeof(SignatureFourierDistributionSampler),
-    "Laplacian Kernel", LaplacianFourierSampler.RegistrationName, "Laplacian")]
+[assembly: LoadableClass(typeof(LaplacianRngGenerator), typeof(LaplacianRngGenerator.Options), typeof(SignatureRngGenerator),
+    "Laplacian Kernel", LaplacianRngGenerator.LoadName, "Laplacian")]
 
 // This is for deserialization from a binary model file.
-[assembly: LoadableClass(typeof(GaussianFourierSampler), null, typeof(SignatureLoadModel),
-    "Gaussian Fourier Sampler Executor", "GaussianSamplerExecutor", GaussianFourierSampler.LoaderSignature)]
+[assembly: LoadableClass(typeof(GaussianRngGenerator.RandomNumberGenerator), null, typeof(SignatureLoadModel),
+    "Gaussian Fourier Sampler Executor", "GaussianSamplerExecutor", GaussianRngGenerator.RandomNumberGenerator.LoaderSignature)]
 
 // This is for deserialization from a binary model file.
-[assembly: LoadableClass(typeof(LaplacianFourierSampler), null, typeof(SignatureLoadModel),
-    "Laplacian Fourier Sampler Executor", "LaplacianSamplerExecutor", LaplacianFourierSampler.LoaderSignature)]
+[assembly: LoadableClass(typeof(LaplacianRngGenerator.RandomNumberGenerator), null, typeof(SignatureLoadModel),
+    "Laplacian Fourier Sampler Executor", "LaplacianSamplerExecutor", LaplacianRngGenerator.RandomNumberGenerator.LoaderSignature)]
 
 // REVIEW: Roll all of this in with the RffTransform.
 namespace Microsoft.ML.Transforms
```

Comment:
I would remove this comment, and change namespace to `Microsoft.ML.Transforms.Projections` to be align with RFF